doc/release-notes.md

```diff
@@ -4,6 +4,45 @@ release-notes at release time)
+- [ZIP 214: Consensus rules for a Zcash Development Fund](https://zips.z.cash/zip-0214)
+- [ZIP 215: Explicitly Defining and Modifying Ed25519 Validation Rules](https://zips.z.cash/zip-0215)
+
+Canopy will activate on testnet at height 950000, and can also be activated at a specific height in regtest mode by setting the config option `-nuparams=0xe9ff75a6:HEIGHT`.
+
+Canopy will activate on mainnet at height 1046400.
+
+See [ZIP 251](https://zips.z.cash/zip-0251) for additional information about the deployment process for Canopy.
+
+Flush witness data to disk only when it's consistent
+-----------------------------------------------------
+This fix prevents the wallet database from getting into an inconsistent state. By flushing witness data to disk from the wallet thread instead of the main thread, we ensure that the on-disk block height is always the same as the witness data height. Previously, the database occasionally got into a state where the latest block height was one ahead of the witness data. This then triggered an assertion failure in `CWallet::IncrementNoteWitnesses()` upon restarting after a zcashd shutdown.
+
+Note that this code change will not automatically repair a data directory that has been affected by this problem; that requires starting zcashd with the `-rescan` or `-reindex` options.
+
+Add DNS seeders
+----------------
+This adds DNS seeders at "zfnd.org" (testnet: testnet.seeder.zfnd.org, mainnet: mainnet.seeder.zfnd.org) and "yolo.money" (testnet: testnet.is.yolo.money, mainnet: mainnet.is.yolo.money) to the list in `chainparams.cpp`.  They're running [CoreDNS](https://coredns.io) with a [Zcash crawler plugin](https://github.com/ZcashFoundation/dnsseeder), the result of a Zcash Foundation in-house development effort to replace zcash-seeder with something memory safe and easier to maintain.
+
+These are validly operated seeders per the existing policy (https://zcash.readthedocs.io/en/latest/rtd_pages/dnsseed_policy.html). For general questions related to either seeder, contact george@zfnd.org or mention @gtank in the Zcash Foundation's Discord. For bug reports, open an issue on the [dnsseeder](https://github.com/ZcashFoundation/dnsseeder) repo.
+
 Changed command-line options
 -----------------------------
 - `-debuglogfile=<file>` can be used to specify an alternative debug logging file.
+
+RPC methods
+------------
+- Add `joinSplitPubKey` and `joinSplitSig` to RPC. These two properties are required to fully reconstruct a Zcash transaction's binary form from the RPC data.
+- Add `estimatedheight` to rpc `getblockchaininfo` response
+
+Initial Block Download
+-----------------------
+- Add a progress bar when in Initial Block Download mode. The progress bar shows both headers (in green) and blocks (in white / inverse of background colour). It is only printed for TTY output. Additionally, the "not mining" message is no longer shown on mainnet, as the built-in CPU miner is not effective at the current network difficulty.
```

Comment:
On my terminal, the headers are in orange/brown and the blocks are in the foreground colour.